<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserRightsEditDialog</class>
 <widget class="QDialog" name="UserRightsEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профиль прав</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Название</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lblRights">
     <property name="text">
      <string>Разрешенные действия</string>
     </property>
     <property name="buddy">
      <cstring>tblRights</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="CInDocTableView" name="tblRights"/>
   </item>
   <item row="2" column="2">
    <widget class="QTreeView" name="tvVisibleGUIElements"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblVisibleGUIElements">
     <property name="text">
      <string>Доступные элементы интерфейса</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtName</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserRightsEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserRightsEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
